### ISSUE TITLE
Music no longer working after H-Encore install

### ISSUE THREAD

**Issue by RadiantAnsel** [TheOfficialFloW/h-encore#42]
So basically this is happening. Music is not working every time H-Encore is installed, if I reboot the Vita without installing H-Encore it playbacks normally, however, the moment I reinstall H-Encore, it wouldn't simply play.
Haven't tested with Videos tho, although music is more important, we can use it while playing, so I hope this gets fixed with some sort of app or plugin.

Thanks!

**Comment by TheOneEyedGrimReaper** [comment]
This problem not with h-encore.
i'm the first h-encore user in hungary and my psvita is working fine with h-encore.
music and all apps working.
yeah and i'm using storagemgr, nonpdrm and oclockvita plugins.

i recommend you restore to factory settings, format memory stick, rebuild database and update to 3.68.
if not working.
you try my h-encore pack

[https://drive.google.com/open?id=1Lih4kFZP13-qFlyCg8XObyUrRWkXdijL](url)

in the h-encore folder you must edit the Make.bat in notepad
where you see YOURCODE
you can just replace with your own vita code, you generated on cma.henkaku.xyz site and save that.

if you're ready run the Make.bat file.
as it is done, you can just copy the PCSG90096 folder to documents/PS Vita/APP/YOURCODE

start qcma -> right click and refresh database.
and you copy that with content manager.

**Comment by RadiantAnsel** [comment]
I don't really think you are the very first User, nor I really care really (Sorry sounding too rude, unleast you are a developer).

I did everything because I seriously think the error was in fact, H-Encore, I mean, music was working before reinstalling H-Encore, and even before the entire existence of it. (I was already 3.68 user)

But I did a lot of things, even installed "Media Importer" which I don't know if it helped or not, but after a reboot I needed to do to install some plugins (PNGShot and ShellSecBat), 

I haven't tested your pack but I'm using the 13MB Compressed H-Encore.

But again, I don't really really know what happened, music worked normally again without a problem, I even rebooted the Vita Once More to be sure and yes, working again.


Thanks anyway.
